docs/index.md

```diff
@@ -1306,6 +1306,11 @@ The following variables are available:
   dataport's name that is a pointer of the type they specified in their CAmkES
   specification. As mentioned previously, the default type is `Buf`.
 
+**_`dataport`_`_size`** (`#include <camkes.h>`)
+
+> If a component has a dataport they will be provided with a symbol _`dataport`_`_size`
```

Comment:
ist not a symbol (ie a (constant) variable that has an address and in known to hte linker) but it's a macro.

Reply:
Improved with 362be5d37e99a84940245abb68a294ac68ababf9